docs/dems.md

```diff
@@ -1,14 +1,19 @@
-## Digital Elevation Models
+# Digital Elevation Models
+Digital Elevation Models are required when processing SAR data to higher-level products, such as the [Radiometric Terrain Correction (RTC)](products.md#rtc) and [Interferometric SAR (InSAR)](products.md#insar) products available [On Demand](https://search.asf.alaska.edu/#/?topic=onDemand) from ASF. 
 
-The quality of the terrain corrections are directly related to the quality of the digital elevation models (DEMs) used in the process of geometrically and radiometrically correcting the SAR imagery. We use DEMs that are publicly available and have wide-ranging coverage. 
+ASF uses DEMs that are publicly available and have wide-ranging coverage. In the past, ASF maintained a collection of DEMs that were pre-processed as appropriate for SAR workflows, and applied a preference hierarchy so that the best available DEM in any given area would be automatically selected for processing. 
```

Comment:
Should we flip this and the following paragraph?  First say "our default is GLO-30"..., then say "in the past, we did something different"?

Reply:
That does make more sense.